### ISSUE TITLE
Make default in GOG Galaxy or create our own updater

### ISSUE THREAD

**Issue by urwrstkn8mare** [GOG-Nebula/.github#6]

**Comment by UncleGoogle** [comment]
Let me mention @gogcom @goggalaxy to inform about work in progress for this initiative. The organization goals are written in the readme https://github.com/orgs/GOG-Nebula/ - please taka a look and let us know if you are interested in cooperation.

We will eventually reach out using official channels too, but lets initiate dev-to-dev talk first.

**Comment by ABaumher** [comment]
At risk of necroing an old thread, has there been any contact through official channels? I'd like to try my hand as well, seeing as Steam is the biggest market and getting their plugin working *should* be a priority. That and i've ~~wasted~~ spent a lot of time working on it

**Comment by urwrstkn8mare** [comment]
@ABaumher Yes please do if you get a response out of them then I think that may be the push needed to get this org moving. I won't lie I don't game much anymore but if there is support from their side I'd be happy to help.

**Comment by UncleGoogle** [comment]
The goal was to have already working alternative for FriendsOfGalaxy. I wanted to rich out GOG again then, so that they dont have to do almost anything.

This issue was the main thing to achieve the goal: https://github.com/GOG-Nebula/.github/issues/4 (which also requires to finish build and release github actions - I recently has worked on those, PR should ready next time I will sit with it).

But as @urwrstkn8mare said feel free to contact and get any information/promise from them so that we could have non zero motivation. Pax

**Comment by ABaumher** [comment]
@AndyGOGGalaxy sorry for mentioning you here but i did say i was going to try to reach out to GOG and it looks like you reached me first. 

I'm just hoping to get the ball rolling here, if we need a better means of discourse that's perfectly fine. If there's someone we should be talking to instead, let us know. I don't mean to put you on the spot, though i guess that's what i'm doing. Sorry. As others mentioned here, we're just trying to get some contact to see where we need to go from here. So if this sort of stuff is ourside of your purview (for lack of better word. It's really late here and i should have been asleep hours ago, so my brain is fried. Work later today is going to be hell), please refer us to whoever's job this actually is. 

@UncleGoogle @urwrstkn8mare @AndrewDWhite

**Comment by urwrstkn8mare** [comment]
@ABaumher Thanks for the effort you've put into fixing the steam integration and also the renewed interest in GOG Nebula.

The knowledge that there is GOG Staff out there (hopefully you get a response about GOG Nebula) and your interest is very motivating. I've got exams right now but nevertheless I'll try to help out as much as I can (especially after). I saw @UncleGoogle has already started work on the release action and I think he's testing it now so thats great. 

Adding you as an org member really won't make much difference as we like to work in pull requests (it might help in development of actions if you want to get involved with that) as much as possible but I think it would be worth it even just for the recognition. But before I go ahead and do things like that I'd like to ask @AndrewDWhite @UncleGoogle @Mixaill for their permission as this is an organisation with no leader.

**Comment by Mixaill** [comment]
I'm fine with it

**Comment by ABaumher** [comment]
There is a pr open on the steam plugin here, just an fyi if you haven't seen it. At this point it's been heavily tested/bulletproofed and is pretty stable, but i should probably mark it as a draft until we get the tests fixed. It should also probably be considered 2.0 by version semantics because there's a breaking change - the public profile fallback is gone (it's no longer necessary).

Exams def take priority, so no worries there.

**Comment by ebonato** [comment]
Maybe pointing to @fastfend, as his last project seems to be GOG Galaxy Store...
